components/CustomerCard.tsx

```diff
@@ -29,6 +29,12 @@ const CustomerCard = ({ name, email, phone }: customer) => {
             placeholder={email}
             readOnly
           />
+          <input
+            type="text"
+            className="input input-readonly"
```

Comment:
in className no need to say "input input-readonly"
Make "input-readonly" components apply "input" from css, example:
`  .input-readonly {
    @apply input cursor-default ...
  }`